.circleci/config.yml

```diff
@@ -44,6 +52,17 @@ jobs:
           name: Running unit tests
           command: ./gradlew --stacktrace test
 
+      - run:
+          name: Waiting for Postgres to be ready
+          command: |
+            for i in `seq 1 10`;
+            do
+              nc -z localhost 5432 && echo Success && exit 0
+              echo -n .
+              sleep 1
+            done
+            echo Failed waiting for Postgres && exit 1
```

Comment:
Error message could have greater clarity.

Id this waiting for Postgres to start? to become responsive?

Reply:
Done.
It's waiting for Postgres to start and be ready accepting requests on its listening port.